src/components/Single/SidePanel/Profile.vue

```diff
@@ -32,6 +32,24 @@
     export default {
         name: "Profile",
         components: {},
+        methods: {
+            followUser() {
+                fetch('http://localhost:3916/api/follower/PostFollowerModel', {
+                    method: 'POST',
+                    headers: new Headers({
+                        'Content-Type': 'application/json'
+                    }),
+                    body: JSON.stringify({
+                        followerId = 1,
+                        followedId = 2
+                    })
+                }).then(result => {
+                    result.json().then(data => {
+                        
+                    });
```

Comment:
What was your plan to do with the result?

Reply:
I was thinking of an popup with added or something. The result will not have any usefull info